pkg/util/i18n/i18n.go

```diff
@@ -53,10 +53,6 @@ func loadSystemLanguage() string {
 		return "default"
 	}
 	pieces := strings.Split(langStr, ".")
-	if len(pieces) == 0 {
```

Comment:
Actually, I think I meant it to be `len(pieces) != 1` to denote: "we expect EN.us" or whatever, and we can't parse this...